geotrek/api/v2/filters.py

```diff
@@ -1,37 +1,45 @@
-from datetime import date
 import operator
+from datetime import date
 from functools import reduce
 
 from coreapi.document import Field
 from django.conf import settings
-from django.db.models.query_utils import Q
 from django.contrib.gis.db.models import Union
+from django.core.exceptions import FieldError
+from django.db.models.query_utils import Q
 from django.utils.translation import ugettext as _
-from rest_framework.filters import BaseFilterBackend
-from rest_framework_gis.filters import InBBOXFilter, DistanceToPointFilter
-
 from geotrek.zoning.models import City, District
+from rest_framework.filters import BaseFilterBackend
+from rest_framework_gis.filters import DistanceToPointFilter, InBBOXFilter
 
 
 class GeotrekQueryParamsFilter(BaseFilterBackend):
     def filter_queryset(self, request, queryset, view):
         return queryset
 
     def get_schema_fields(self, view):
-        field_dim = Field(name='dim', required=False,
-                          description=_('Set geometry dimension (2 by default for 2D, 3 for 3D)'),
-                          example=3, type='integer')
         field_language = Field(name='language', required=False,
                                description=_("Set language for translation. 'all' by default"),
                                example="fr")
         field_format = Field(name='format', required=False,
-                             description=_("Set output format (json / geojson). JSON by default"),
+                             description=_("Set output format (json / geojson). JSON by default. Please note taht only the endpoints containing geographical informations can handle the geojson format."),
```

Comment:
that

Reply:
Est-ce qu'il ne faut pas le déplacer dans GeotrekInBBoxFilter quitte à renommer ce dernier en GeotrekGeometryFilter ?